ddclient.in

```diff
@@ -5637,6 +5649,83 @@ sub nic_googledomains_update {
     }
 }
 
+######################################################################
+## nic_mythicdyn_examples
+##
+## written by Reuben Thomas
+##
+######################################################################
+sub nic_mythicdyn_examples {
+    return <<"EoEXAMPLE";
+o 'mythicdyn'
+
+The 'mythicdyn' protocol is used by the Dynamic DNS service offered by
+www.mythic-beasts.com.
+
+Configuration variables applicable to the 'mythicdyn' protocol are:
+  protocol=mythicdyn           ##
+  ipv6=no|yes                  ## whether to set an A record (default, ipv6=no)
+                               ## or AAAA record (ipv6=yes).
+  login=service-login          ## the user name provided by the admin interface
+  password=service-password    ## the password provided by the admin interface
+  fully.qualified.host         ## the host registered with the service
+
+Note: this service automatically sets the IP address to that from which the
+request comes, so the IP address detected by ddclient is only used to keep
+track of when it needs updating.
+
+Example ${program}.conf file entries:
+  ## Single host update.
+  protocol=mythicdyn,                       \\
+  login=service-login                       \\
+  password=service-password,                \\
+  host.example.com
+
+  ## Multiple host update.
+  protocol=mythicdyn,                       \\
+  login=service-login                       \\
+  password=service-password,                \\
+  hosta.example.com,hostb.sub.example.com
+EoEXAMPLE
+}
+######################################################################
+## nic_mythicdyn_update
+######################################################################
+sub nic_mythicdyn_update {
+    debug("\nnic_mythicdyn_update --------------------");
+
+    # Update each set configured host.
+    foreach my $h (@_) {
+        info("%s -- Setting IP address.", $h);
+        verbose("UPDATE:", "updating %s", $h);
```

Comment:
```suggestion
```
This is duplicated

Reply:
I'm not the expert here, but the general format seems to follow other backends, it's just that the "UPDATE" line is easy to parse out (I presume). For example, in lines 5493-4, the `nic_godaddy_update` subroutine outputs duplicate information on two successive lines, one a call to `info` and one to `verbose`.

Reply:
There is a general inconsistency in the entire code how provider are handled.

Reply:
Fine, I've removed this line.